lint.sh

```diff
@@ -1,6 +1,24 @@
 #!/bin/bash
 
 rc=0
+venv=.lintvenv
+# exclude venv and git dir from linting
+fargs=(. -not -path \*${venv}\* -not -path \*.git\*)
+
+trap cleanup EXIT
+cleanup(){
+  deactivate
```

Comment:
In the build that ran, this line isn't executing properly because of `RPC_GATING_LINT_USE_VENV=no`.

https://rpc.jenkins.cit.rackspace.net/job/rcbops/job/rpc-gating/job/PR-58/1/console
`./lint.sh: line 10: deactivate: command not found`

What's the reasoning behind not running the rpc-gating PR builds in a virtualenv?